### DISCUSSION TITLE
Welcome to btczee Discussions!

### DISCUSSION THREAD

**Discussion by AbdelStark** [zig-bitcoin/btczee#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Prabhat1308** [comment]
Hi everyone ! I am Prabhat , an undergrad from IIT Roorkee . I do OS part-time and write Rust for a living. I am a Summer of Bitcoin'24 fellow in Bitcoin-Core organisation. Although a zig newbie , interested in learning it and what better way to learn it by implementing bitcoin-core using it. Expecting to add some value by contributing to the project !

**Reply by AbdelStark** [reply]
Wonderful! Welcome mate. Looking forward to your contributions!
